vendor/github.com/google/cel-go/interpreter/functions/functions.go

```diff
@@ -45,6 +45,10 @@ type Overload struct {
 	// Function defines the overload with a FunctionOp implementation. May be
 	// nil.
 	Function FunctionOp
+
+	// NonStrict specifies whether the Overload will tolerate arguments that
+	// are types.Err or types.Unknown.
+	NonStrict bool
```

Comment:
do we need to sweep our added library methods to consider whether we should set this on any of them?

Reply:
I don't really understand the implications enough to know either way... if this is defaulting to current behavior and we don't want any of our library methods to evaluate Err or Unknown args, this probably seems ok

Reply:
This feature should not be used except in very rare circumstances. None of the functions currently enabled for kubernetes would really benefit from using this feature, unless you're introducing a function like `try`, e.g. `try(a.b.c, "default")`, and even that would be better served by using a strict function which more deliberately checks whether `a` has property `b` and `b` has property `c`.